src/pages/Address/AddressComp.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { ReactNode, useContext, useState } from 'react'
+import Pagination from '../../components/Pagination'
+import HelpIcon from '../../assets/qa_help.png'
+import OverviewCard, { OverviewItemData } from '../../components/Card/OverviewCard'
+import TitleCard from '../../components/Card/TitleCard'
+import Tooltip from '../../components/Tooltip'
+import TransactionItem from '../../components/TransactionItem/index'
+import { AppContext } from '../../contexts/providers/index'
+import i18n from '../../utils/i18n'
+import { localeNumberString } from '../../utils/number'
+import { isMobile } from '../../utils/screen'
+import { startEndEllipsis } from '../../utils/string'
+import { shannonToCkb } from '../../utils/util'
+import {
+  AddressLockScriptItemPanel,
+  AddressLockScriptPanel,
+  AddressPendingRewardTitlePanel,
+  AddressTransactionsPagition,
+  AddressTransactionsPanel,
+} from './styled'
+import browserHistory from '../../routes/history'
+
+const addressContent = (address: State.Address) => {
+  const addressText = isMobile() ? startEndEllipsis(address.addressHash, 10) : address.addressHash
+  return address.addressHash ? addressText : i18n.t('address.unable_decode_address')
+}
+
+const AddressPendingRewardTitle = () => {
+  const [show, setShow] = useState(false)
+  return (
+    <AddressPendingRewardTitlePanel>
+      {`${i18n.t('address.pending_reward')}`}
+      <div
+        id="address__pending_reward_help"
+        tabIndex={-1}
+        onFocus={() => {}}
+        onMouseOver={() => {
+          setShow(true)
```

Comment:
I'm not quite sure but is it possible to use CSS to show the tooltip?

Reply:
We can leave it now.

Reply:
Tooltip works well on PC, but we need to add click event on mobile and remove tooltip when focus leave. 

So I add `onMouseOver` event.  @Keith-CY

Reply:
OK, I see.